backend/src/main/java/meetup/connect/config/ApplicationConfig.java

```diff
@@ -0,0 +1,49 @@
+package meetup.connect.config;
+
+import meetup.connect.user.UserService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.config.annotation.authentication.configuration.AuthenticationConfiguration;
+
+@Configuration
+public class ApplicationConfig {
```

Comment:
nit: I don't see a particular reason, why it's separated from `SecurityConfig`, you're configuring security in `ApplicationConfig`

Reply:
You could easily create a configuration class that contains all `@Bean` configurations.